src/rprblender/nodes/blender_nodes.py

```diff
@@ -1428,7 +1428,8 @@ class ShaderNodeMath(NodeParser):
 
     def export(self):
         op = self.node.operation
-        in1 = self.get_input_value(0)
+        # we use a value data from the input
+        in1 = self.get_input_value(0).rgb_to_hsv().get_channel(2)
```

Comment:
Compare this with cycles. This has to be checked.

Reply:
reverted https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/pull/595/commits/552598c82734de2878afcb3acf6b3d8d0cc5efb0